src/main/java/codeu/enumeration/ActivityTypeEnum.java

```diff
@@ -0,0 +1,7 @@
+package codeu.enumeration;
```

Comment:
What is this exactly? The enumeration. How does it connect to the code?

I just wanna know because I don't know lol.

Reply:
Good question! The `codeu.enumeration` package is a subpackage of the app I just created to hold enums in and keep everything organized c:

Reply:
As for the ActivityTypeEnun itself, it lists (enumerates) what values an Activity's `type` property can take on

Reply:
What kind of values? Is it gonna be like different properties of the activity feed? Or the different types of activities the activity feed can show/represent?

Reply:
Exactly the latter c:

Reply:
That's pretty cool!
https://media.giphy.com/media/uJSlAuoxMbl0A/giphy.gif

Reply:
Thanks for giving more detail! @moustafaelsisy @NakedOtaku